vnext/local-cli/runWindows/utils/deploy.js

```diff
@@ -218,7 +218,10 @@ async function deployToDesktop(options, verbose, slnFile) {
     );
   }
 
-  let args = ['--remote-debugging', options.proxy ? 'true' : 'false'];
+  let args = [];
```

Comment:
this is the actual change, the rest is yarn autoformatting

Reply:
Any reason we didn't keep using --remote-debugging as the process arg?  From an apps perspective I find --remote-debugging to have a more obvious meaning than --proxy.  -- And this is something that will likely be in app code to support the CLI going forward.

Reply:
--proxy was an existing flag that had no effect. --remote-debugging was not used by any cpp code or mentioned in the prop list for run-windows.
Frankly, I just followed what the task asked.

Reply:
I was just thinking the actual cmdline arg that gets send to the app, rather than the CLI arg on run-windows.  But looking at it further, I don't see a similar option in iOS or android's CLI.  This code also looks like a potential security bug to me.  This would allow any app shipping with the default RNW template to be run with in a mode where its JS would be provided from some other source. I think the native handling app of the cmdline arg should be debug only?

Reply:
@rectified95 I know it's outside of the scope of this PR, but could you make the `--direct-debugging` flag debug only as well (since it provides similar functionality)?

Reply:
IIUC --proxy came from the old C# implementation.  I also feel --remote-debugging would be a better name for this option, even though it's technically a behavior change.  @jonthysell any objection to renaming?

Reply:
+1 on using something that has debug in the name :)

Reply:
I have no problem with the renaming. The request for `--proxy` was from 2018 for the "rnw current".

Reply:
Renamed the parameter, and made both the direct/remote debugging options available only in debug mode.